src/assets/scripts/client/airline/AirlineController.js

```diff
@@ -39,6 +54,7 @@ export default class AirlineController {
      * @param icao {string}
      */
     addAirline(icao) {
+        console.error('.addAirline() is a deprecated method');
```

Comment:
When doing this sort of thing, do you typically want to add a TODO and get it torn out later down the road?